### PR TITLE
Setup Popup Window for Work Section

### PR DESCRIPTION
**Pull request by chelobotix**
In this PR, I fulfilled the following tasks:
- [x] Create a Popup window for mobile and desktop versions.
- [x] Add a JavaScript array to store all the information for the projects.
- [x] Update the main page so that the projects section is created dynamically from a JS script
- [ x] Update the README file with the new feature

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -113,158 +114,7 @@ <h1>
...
-              </div>
-            </ul>
-            <div class="div_button">
-              <button class="btn_see_projects btn_effects" type="button" href="#">See Projects</button>
-            </div>
-          </div>
-        </article>
-      </div>
-    </section>
+    <section class="section_works" id="Portoflio"></section>
```
Comment (huseyinbiyik) [inline index.html:117]: Kindly fix the width of the modal as per the given Figma template.
Your design:

Screenshot 2023-01-06 at 1 29 48 AM

Figma template:
Screenshot 2023-01-06 at 1 30 18 AM

File: popup.js [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+
+  cardRow.classList.add('card_row');
+  placeholder.classList.add('placeholder');
+  cardContainer1.classList.add('card_container1');
+  buttonDiv.classList.add('div_button');
+  button.classList.add('btn_see_projects');
+  button.classList.add('btn_effects');
+  button.id = recentWorksObj[i].id;
+
+  if (window.matchMedia('(max-width: 767px)').matches) {
```
Comment (huseyinbiyik) [inline popup.js:93]: Please fix the height of the modal on the mobile version. It uses almost the half of the page as seen below.

Figma template:
Screenshot 2023-01-06 at 1 37 42 AM
Screenshot 2023-01-06 at 1 38 19 AM

File: index.html [hunk 3]
```diff
@@ -445,8 +295,32 @@ <h3 class="heading_footer">marceloalarconbarrenechea@gmail.com</h3>
...
+    <!-- Modal -->
+    <div class="blur invisible">
+      <div class="modal-project">
+        <div class="close"></div>
+        <img class="modal-project" src="" alt="Project Picture" />
+        <div class="dsk-pos">
+          <h3 class="modal-project"></h3>
+          <div class="modal-btn-div modal-dsk">
+            <button class="modal-btn1 btn_effects">See Live<i class="bi bi-box-arrow-up-right"></i></i></button>
+            <button class="modal-btn2 btn_effects">See Source<i class="bi bi-github"></i></button>
```
Comment (huseyinbiyik) [inline index.html]: Please fix the icon like on the Figma template. The icons is not under the link, it is just located on the next to it.
Screenshot 2023-01-06 at 1 33 15 AM
Screenshot 2023-01-06 at 1 33 26 AM

File: index.html (hunk 1 above)
Comment (huseyinbiyik) [inline index.html:117]: Please make sure the Lorem Ipsum text is not overflowing from the modal and there is padding on the bottom of the text.
Screenshot 2023-01-06 at 1 30 55 AM
Figma template:
Screenshot 2023-01-06 at 1 31 17 AM

**Review by huseyinbiyik — CHANGES_REQUESTED** [review]
Hi @chelobotix and @adelphinsucasa,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
Almost there

## Highlights
- Works section created dynamically with JavaScript 👌

## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
- [OPTIONAL] Please fix the license link on the Readme file. 

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by chelobotix** [comment]
Observation 1: Kindly fix the width of the modal as per the given Figma template.

image

Observation 2: Please fix the height of the modal on the mobile version. It uses almost half of the page as seen below.

image

image

Observation 3: Please fix the icon like on the Figma template. The icons is not under the link, it is just located on the next to it.

image

Observation 4: Please make sure the Lorem Ipsum text is not overflowing from the modal and there is padding on the bottom of the text.

image

**Review by BregornOriginal — APPROVED** [review]
## Hi @chelobotix 

## Your project is complete!  There is nothing else to say other than... it's time to merge it 

## Congratulations!

tenor-1

## Highlights

✅ Good commit history
✅ Good work implementing the changes from the previous reviewer

## Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.
Cheers and Happy coding!:aplauso::aplauso::aplauso:
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, remember to tag me in your question so I can receive the notification.
------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)[.](https://airtable.com/shrQAqnBwek5a0O0s)._)

**Pull request merged**